backend/clap/proto/data_store.proto

```diff
@@ -0,0 +1,106 @@
+// Proto file from the shared folder
+syntax = "proto3";
+
+package data_store;
+
+option java_multiple_files = true;
+option java_outer_classname = "DataStoreProto";
+option go_package = "github.com/Veermove/zpi/data-store";
+
+enum ProjectState {
+    DRAFT = 0;
+    SUBMITTED = 1;
+    ACCEPTED = 2;
+    REJECTED = 3;;
+}
+
+enum RatingType {
+    INDIVIDUAL = 0;
+    INITIAL = 1;
+    FINAL = 2;
+}
+
+message SubmissionRequest {
+    int32 contest_id = 1;
```

Comment:
This should take assessorId, instead of contest id. We need to retrieve all submissions assigned to a given assessor (both archived and active)